### ISSUE TITLE
java.util.MissingResourceException occurs

### ISSUE THREAD

**Issue by junnohta999** [DoqueDB/doquedb#12]
dqunload shows a stack trace which means some resource data is absent.
```shell
$ dqunload unload_script.txt root doqadmin
java.util.MissingResourceException: Can't find bundle for base name jp.co.ricoh.sydney.admin.util.resource.version, locale ja_JP
        at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2045)
        at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1683)
        at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1586)
        at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1549)
        at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:858)
        at jp.co.ricoh.sydney.admin.util.Res.getVersion(Res.java:53)
        at jp.co.ricoh.sydney.admin.unload.unload.main(unload.java:102)
4月 09, 2024 1:51:04 午後 jp.co.ricoh.sydney.admin.util.log.Log info
情報: Version 0.0.0
...
```

**Comment by htozaki** [comment]
It seems necessary file  `jp/co/ricoh/sydney/admin/util/resource/version.properties` is missing in `bin/java/unload.jar`.

**Comment by htozaki** [comment]
Pull request to fix it: https://github.com/DoqueDB/doquedb/pull/13

**Comment by htozaki** [comment]
Solved. However unload failed unless LANG environment is set to ja_JP.

> java.lang.ClassNotFoundException: jp.co.ricoh.doqeudb.jdbc.Driver
>         at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
>         at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
>         at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
>         at java.base/java.lang.Class.forName0(Native Method)
>         at java.base/java.lang.Class.forName(Class.java:315)
>         at jp.co.ricoh.sydney.admin.unload.unload.open(unload.java:279)
>         at jp.co.ricoh.sydney.admin.unload.unload.main(unload.java:106)

**Comment by htozaki** [comment]
Fix typos in resources: https://github.com/DoqueDB/doquedb/pull/18

**Comment by htozaki** [comment]
released in v1.1.1.
